core/modules/savers/put.js

```diff
@@ -20,7 +20,7 @@ Retrieve ETag if available
 */
 var retrieveETag = function(self) {
 	var headers = {
-		Accept: "*/*;charset=UTF-8"
+		Accept: "*/*"
```

Comment:
On line 52 the OPTIONS request doesn't set a header explicitly, so perhaps it would be okay to remove the header entirely here too, but I'm not confident there isn't a reason why it's needed, so I figure let's keep it in to minimize the change and the risk of a regression.